openBCIBoard.js

```diff
@@ -779,8 +917,10 @@ function OpenBCIFactory() {
                         console.log('\tNot calculating impedance for either P and N input.');
                     }
                 }
+                if(pInput) this.impedanceArray[channelNumber - 1].P.raw = this.impedanceTest.impedanceForChannel;
+                if(nInput) this.impedanceArray[channelNumber - 1].N.raw = this.impedanceTest.impedanceForChannel;
                 resolve(channelNumber);
-            }, 250);
+            }, 1000);
```

Comment:
This is how long we observe the incoming channel for, when testing it appeared that the sudden injection of current needed some time to stabilize.